src/data_provider/src/sysInfoLinux.cpp

```diff
@@ -438,6 +552,35 @@ nlohmann::json SysInfo::getPorts() const
         }
     }
 
+    std::vector<int64_t> inodes;
```

Comment:
In the way is used this variable, a type std::deque would have better performance.

Reply:
I thought that the benefit of deque was in the front insert. But I verified that there's an improvement in this case too.